src/compile/data/index.ts

```diff
@@ -29,4 +29,9 @@ export interface DataComponent {
    * For facets, we store the reference to the root node.
    */
   facetRoot?: FacetNode;
+
+  /**
+   * Parse properties passed down from ancestors.
+   */
+  modelParse: Dict<string>;
```

Comment:
modelParse is a pretty confusing name -- it gives me no clue what it means when I first read the code.   maybe `ancestorParse`?